nomos-services/network/src/backends/libp2p.rs

```diff
@@ -145,9 +155,7 @@ impl NetworkBackend for Libp2p {
                             Command::Broadcast { topic, message } => {
                                 tracing::debug!("sending message to mixnet client");
                                 let msg = MixnetMessage { topic, message };
-                                // TODO: use `wire` instead of json, by resolving import cycles
-                                let msg = serde_json::to_vec(&msg).unwrap();
-                                log_error!(mixnet_client.send(msg, std::time::Duration::ZERO));
+                                log_error!(mixnet_client.send(msg.as_bytes().to_vec(), std::time::Duration::ZERO));
```

Comment:
Why `to_vec`? is a reference not enough?

Reply:
You're right. I simplified this: d4fffa3